backends/gstreamer/src/lib.rs

```diff
@@ -80,6 +80,25 @@ impl Backend for GStreamerBackend {
     fn create_videoinput_stream(&self, set: MediaTrackConstraintSet) -> Option<Box<MediaStream>> {
         media_capture::create_videoinput_stream(set).map(|s| Box::new(s) as Box<MediaStream>)
     }
+
+    fn can_play_type(&self, media_type: &str) -> bool {
+        match gst::caps::Caps::from_string(media_type) {
+            Some(caps) => {
+                let factories = gst::ElementFactory::list_get_elements(
+                    gst_ffi::GST_ELEMENT_FACTORY_TYPE_ANY,
```

Comment:
I was not sure what to pass here. Should this parameter get passed as an argument?